src/app.js

```diff
@@ -1 +1,41 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { calculateBullsAndCows } = require('./calculateBullsAndCows');
+const { generateNumber } = require('./generateNumber');
+const { terminal } = require('./terminal');
+
+console.log('Welcome to Bulls and Cows game!');
+
+const secretNumber = generateNumber();
+let attempts = 0;
+
+function play() {
+  terminal.question('Enter your guess (4 different digits): ', guess => {
+    attempts++;
+
+    if (!isValidNumber(guess)) {
+      // eslint-disable-next-line no-console
+      console.log('Wrong data, please enter valid number');
+
+      return play();
+    }
+
+    const { bulls, cows } = calculateBullsAndCows(secretNumber, guess);
+
+    if (bulls === 4) {
```

Comment:
When you use some static values like 4 here especially when you use it in two or more places, it's better to move it to some variable.